src/test/unit/resources/test_detectors.py

```diff
@@ -0,0 +1,112 @@
+import json
+import os
+import urllib.request
+from contextlib import contextmanager
+
+from opentelemetry.sdk import resources
+from opentelemetry.semconv.resource import ResourceAttributes
+
+from lumigo_opentelemetry.resources.detectors import (
+    ProcessResourceDetector,
+    LumigoDistroDetector,
+    EnvVarsDetector,
+    get_resource,
+)
+
+
+def test_process_detector():
```

Comment:
@nirLumigo we also need to test that the secrets scrubbing works correctly for env vars.

Reply:
Added at: [5022872](https://github.com/lumigo-io/opentelemetry-python-distro/pull/44/commits/5022872a5deb7283d135a2951f093685e650921c)